services/pom.xml

```diff
@@ -164,6 +164,12 @@
       <groupId>io.springfox</groupId>
       <artifactId>springfox-swagger2</artifactId>
     </dependency>
+    <dependency>
```

Comment:
I am not sure, but it seems that we sometimes mock web services from there, i.e. when test, and that in this case the web module dependencies are not loaded...
(nibor deps in test scope, i.e. not transitive, anywhere but in web ?)